x-pack/plugins/aiops/public/components/log_categorization/show_flyout.tsx

```diff
@@ -12,11 +12,8 @@ import { pick } from 'lodash';
 import type { CoreStart } from '@kbn/core/public';
 import { Storage } from '@kbn/kibana-utils-plugin/public';
 
-import {
-  toMountPoint,
-  wrapWithTheme,
-  KibanaContextProvider,
-} from '@kbn/kibana-react-plugin/public';
+import { wrapWithTheme, KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
```

Comment:
```suggestion
import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
```

Reply:
`wrapWithTheme` is still needed as it needs to be passed to the `DatePickerContextProvider` component.

Reply:
It's no longer needed after this change https://github.com/elastic/kibana/commit/9b8e1e65e7960dc72524e2b8b8023c41dd9472d2